src/Components/ProjectList/index.js

```diff
@@ -38,6 +39,7 @@ export default class ProjectList extends React.Component {
 
   renderTitles = () => {
     return <tr>
+        <th>#</th>
```

Comment:
```suggestion
        <th>ID</th>
```
As we mention adding a user to a certain project ID, makes it a bit clearer if we label the number as ID here as well.